Vagrantfile

```diff
@@ -102,8 +101,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     configure_ansible_provisioning(vm_config)
   end
 
+  config.vm.define("el9", primary: true) do |vm_config|
+    vm_config.vm.box = "generic/centos9s"
+    vm_config.vm.host_name = "candlepin-el9.example.com"
+
+    # Uncomment these lines for forward the Candlepin standard dev ports to this guest.
+    # vm_config.vm.networking "forwarded_port", protocol: "tcp", guest: 8080, host: 8080
+    # vm_config.vm.networking "forwarded_port", protocol: "tcp", guest: 8443, host: 8443
+
+    vm_config.vm.provision "shell", inline: "dnf update -y dnf ca-certificates"
+    configure_ansible_provisioning(vm_config)
+  end
+
   config.vm.define("f35", autostart: false) do |vm_config|
-    vm_config.vm.box = "fedora/35-cloud-base"
+    vm_config.vm.box = "generic/fedora35"
```

Comment:
What is critical about the boxes themselves that we need to move away from the official boxes provided by the projects to the third-party autogenerated ones?

I get that we don't have options for an official centos9 box at the moment, but for the others, what are we gaining and/or losing by switching boxes here? What risks come from having outdated boxes that isn't resolved by normal in-VM updates of the OS?